### DISCUSSION TITLE
[PostgreSQL error] failed to retrieve PostgreSQL server_version_num: connection refused

### DISCUSSION THREAD

**Discussion by nboddu009** [Kong/kong#13009] [Help]
Hi Team,
 Can someone help with installation issue in Ubuntu, I had installed Kong but when i  am trying to start the kong, It is giving connection refused issue. 
I had used [https://docs.konghq.com/gateway/3.6.x/install/linux/ubuntu/](url) to install the kong.

user1$ sudo kong start
2024/05/09 22:22:55 [warn] ulimit is currently set to "1024". For better performance set it to at least "4096" using "ulimit -n"
2024/05/09 22:22:55 [warn] ulimit is currently set to "1024". For better performance set it to at least "4096" using "ulimit -n"
Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: connection refused

  Run with --v (verbose) or --vv (debug) for more details
user1$ psql --version
psql (PostgreSQL) 14.12 (Ubuntu 14.12-1.pgdg20.04+1)

**Comment by samugi** [comment]
duplicate of https://github.com/Kong/kong/issues/13010
